packages/schema/.configu.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "https://raw.githubusercontent.com/configu/configu/main/packages/schema/.configu.json",
+  "$id": "https://api.configu.com/schema/.configu.json",
```

Comment:
Are these schemas planned to be served via the platform API? Currently, the API doesn't serve these schemas.

Reply:
Thanks that was a bug!